### PR TITLE
For #1423: Add Sticky decorator for Text

### PR DESCRIPTION
**Pull request by andreoss**
- Add `Sticky` implementation for `Text`
- This is for #1423

### CODE REVIEW

**Comment by victornoel** [comment]
@0crat status

**Comment by 0crat** [comment]
> @0crat status ([here](https://github.com/yegor256/cactoos/issues/1429#issuecomment-679826994))

@victornoel This is what I know about this job in C63314D6Z, as in [§32](http://www.zerocracy.com/policy.html#32):

  * The job [#1429](https://github.com/yegor256/cactoos/issues/1429) is [in scope](http://www.zerocracy.com/policy.html#14) for 9hr
  * The role is `REV`
  * The job is assigned to @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu) for [9hr](http://www.zerocracy.com/policy.html#8)
  * There is no monetary reward attached, it's a [free](http://www.zerocracy.com/policy.html#2) job
  * The job doesn't have any [impediments](http://www.zerocracy.com/policy.html#9)
  * The [budget](http://www.zerocracy.com/policy.html#4) is 15 minutes/points
  * These users are banned and won't be assigned:
    * @andreoss[/z](https://www.0crat.com/u/andreoss): This user reported the ticket
  * Job [footprint](https://www.0crat.com/footprint/C63314D6Z?q=%7Bjob%3A%27gh%3Ayegor256%2Fcactoos%231429%27%2Ctype%3A%7B%24not%3A%2F%5ENotify%2F%7D%7D) (restricted area)

**Comment by victornoel** [comment]
@marceloamadeu here too you have been assigned

**Comment by marceloamadeu** [comment]
@victornoel looks good to me

**Comment by victornoel** [comment]
@andreoss please see my comment in #1426, let's clarify the expectation there before rushing to implement it please :)

**Comment by victornoel** [comment]
@marceloamadeu please try to find problems in PRs in the future, there were many problems to find, starting with the fact that the issue it solves is not in scope, and that some of the implementation didn't comply with what was discussed in the issue :)

**Comment by marceloamadeu** [comment]
> @marceloamadeu please try to find problems in PRs in the future, there were many problems to find, starting with the fact that the issue it solves is not in scope, and that some of the implementation didn't comply with what was discussed in the issue :)

@victornoel Ok...

**Comment by victornoel** [comment]
> @andreoss please see my comment in #1426, let's clarify the expectation there before rushing to implement it please :)

@andreoss sorry, I meant #1423 of course :)

File: src/main/java/org/cactoos/text/Sticky.java [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+     */
+    public Sticky(final Text txt) {
+        this((Scalar<String>) txt::asString);
+    }
+
+    /**
+     * Private ctor.
+     * @param sclr Scalar to cache
+     */
+    private Sticky(final Scalar<String> sclr) {
```
Comment (victornoel) [inline src/main/java/org/cactoos/text/Sticky.java:58]: @andreoss please remove this constructor and just keep the `Text` one

**Review by victornoel — CHANGES_REQUESTED** [review]
@andreoss just one comment

File: src/main/java/org/cactoos/text/Sticky.java (hunk 1 above)
Comment (andreoss) [reply src/main/java/org/cactoos/text/Sticky.java]: @victornoel Done

**Comment by victornoel** [comment]
@andreoss thank you

**Comment by victornoel** [comment]
@rultor merge

**Comment by 0crat** [comment]
Code review was too long (10 days), architects (@victornoel) were penalized, see [§55](http://www.zerocracy.com/policy.html#55)

**Comment by 0crat** [comment]
@sereshqua[/z](https://www.0crat.com/u/sereshqua) please review this job completed by @marceloamadeu[/z](https://www.0crat.com/u/marceloamadeu), as in [§30](http://www.zerocracy.com/policy.html#30); the job will be fully closed and all payments will be made when the quality review is completed

**Comment by sereshqua** [comment]
@0crat quality bad

**Comment by 0crat** [comment]
Quality is low, no payment, see [§31](http://www.zerocracy.com/policy.html#31)

**Pull request merged**